app/Http/Controllers/TagController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\TagRequest;
+use App\Http\Resources\TagResource;
+use App\Models\Tag;
+use Illuminate\Http\JsonResponse;
+
+class TagController extends Controller
+{
+    public function store(TagRequest $request): TagResource
+    {
+        $tag = Tag::query()->create($request->validated());
```

Comment:
Can we make controller like here? 

https://github.com/blumilksoftware/interns2025c/pull/30/files#diff-e74a1e9d4e2d6366d6b69b0918e117bc9d1459bc746658131eeea75046f60783